README.md

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
the changes in this file are just changing repo strings from old ipfs-shipyard ones: `s/ipfs-shipyard\/ipfs-desktop/ipfs\/ipfs-desktop/`